paddle/fluid/operators/batch_norm_op.cc

```diff
@@ -576,6 +617,7 @@ std::unique_ptr<framework::OpDesc> BatchNormGradMaker::Apply() const {
   auto *op = new framework::OpDesc();
   op->SetType(GradOpType());
   op->SetInput("X", Input("X"));
+  op->SetInput("Y", Output("Y"));
```

Comment:
Do not add "Y" in grad op desc. It would increase memory usages of models.

Reply:
Y cannot be found without this line.

Reply:
I agree not to add `Y`. There is no need Y for normal BN. Only need for Inplace-ABN.